### PR TITLE
Update/0.10.5

### PR DESCRIPTION
**Pull request by Dominion5254**
- Add postgresql-client-14 to Dockerfile (LNBits backups)
- Bump version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,11 @@ RUN wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/d
 RUN curl -sSL https://install.python-poetry.org | python3 -
 ENV PATH="/root/.local/bin:$PATH"
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt bullseye-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+RUN apt-get update
```
Comment (islandbitcoin) [inline Dockerfile:14]: you already have apt-get update on line 8, might not need it here again.

**Review by islandbitcoin — DISMISSED** [review]
small change, looks good otherwise

File: Dockerfile [hunk 2]
```diff
@@ -10,6 +10,11 @@ RUN wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/latest/d
 RUN curl -sSL https://install.python-poetry.org | python3 -
 ENV PATH="/root/.local/bin:$PATH"
 
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt bullseye-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
```
Comment (k0gen) [inline Dockerfile:13]: move line 13,14 just under line 7 and just add postgresql-client-14 to list of packages that are needed

Comment (k0gen) [reply Dockerfile:13]: `gcc` and `make` is not needed `build-essential` is adding those

**Review by k0gen — CHANGES_REQUESTED** [review]
Integrate

File: Dockerfile (hunk 1 above)
Comment (Dominion5254) [reply Dockerfile:14]: It seems this apt-get update on line 15 is also required as removing line 15 results in an error finding the package.

Upstream also includes both `apt-get update` commands

```
#0 1.466 E: Unable to locate package postgresql-client-14
------
Dockerfile:15
--------------------
  13 |     RUN echo "deb http://apt.postgresql.org/pub/repos/apt bullseye-pgdg main" > /etc/apt/sources.list.d/pgdg.list
  14 |     RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
  15 | >>> RUN apt-get install -y postgresql-client-14
  16 |     
  17 |     WORKDIR /app/
--------------------
ERROR: failed to solve: process "/bin/sh -c apt-get install -y postgresql-client-14" did not complete successfully: exit code: 100
```

**Review by k0gen — APPROVED** [review]
ok

**Pull request merged**